src/pybind/mgr/dashboard/services/ceph_service.py

```diff
@@ -174,6 +174,10 @@ def get_rates(cls, svc_type, svc_name, path):
         :return: the derivative of mgr.get_counter()
         :rtype: list[tuple[int, float]]"""
         data = mgr.get_counter(svc_type, svc_name, path)[path]
+        return cls.get_rates_from_data(data)
+
+    @classmethod
+    def get_rates_from_data(cls, data):
```

Comment:
cool, can you please move this into `mgr_util.py`? This is a function that really wants to be used by other modules as well.